src/main/java/reposense/parser/SinceDateArgumentType.java

```diff
@@ -16,14 +17,17 @@
 public class SinceDateArgumentType extends DateArgumentType {
     /*
      * When user specifies "d1", arbitrary first commit date will be returned.
+     * This date is equivalent to 1970-01-01 00:00:00 in UTC time.
      * Then, ReportGenerator will replace the arbitrary since date with the earliest commit date.
      */
-    public static final LocalDateTime ARBITRARY_FIRST_COMMIT_DATE = LocalDateTime.ofInstant(
-            Instant.ofEpochMilli(Long.MIN_VALUE), ZoneId.of("Z"));
     public static final String FIRST_COMMIT_DATE_SHORTHAND = "d1";
+    private static final ZonedDateTime ARBITRARY_FIRST_COMMIT_DATE = ZonedDateTime.ofInstant(
+            Instant.ofEpochMilli(0), ZoneId.of("Z"));
+    private static final LocalDateTime ARBITRARY_FIRST_COMMIT_DATE_UTC_LOCAL = ARBITRARY_FIRST_COMMIT_DATE
```

Comment:
The variable naming seems a little confusing. May I suggest using `ARBITRARY_FIRST_COMMIT_DATE_UTC` and `ARBITRARY_FIRST_COMMIT_DATE_LOCAL` instead?